api/src/main/java/org/jboss/forge/roaster/model/Type.java

```diff
@@ -34,6 +38,8 @@
 
    boolean isQualified();
 
+   boolean isNameQualified();
```

Comment:
I'd be ok with it. The current implementation of isQualified relies on that internal implementation choice